homeassistant/components/hardware/models.py

```diff
@@ -17,12 +17,24 @@ class BoardInfo:
     revision: str | None
 
 
+@dataclass
+class USBInfo:
+    """USB info type."""
+
+    vid: str
+    pid: str
+    serial_number: str | None
+    manufacturer: str | None
+    description: str | None
```

Comment:
Maybe we want to send some additional data, such as friendly name of the dongle and if the dongle is currently connected or not.

Reply:
Let's start simple.